build-aux/snap/snapcraft.yaml

```diff
@@ -206,12 +218,20 @@ parts:
       if [ "${VERSION/+fips/}" != "$VERSION" ] ; then
           # we have a '+fips' element in the version, which may be coming from
           # debian/changelog or git tag
-          echo "-- detected FIPS build"
-          touch fips-build
+          echo "-- detected FIPS build from version $VERSION"
+          touch "${CRAFT_PROJECT_DIR}/fips-build"
```

Comment:
I do not like that much that pull modifies `$CRAFT_PROJECT_DIR`. It is supposed to modify `$CRAFT_PART_SRC`.

Reply:
Could we install the files in the stage? So that libcrypto-fips can see them?

Reply:
If the file was to be placed in stage, wouldn't it be picked up by default since there's no stage for snapd?

I suppose I can simply run the `is-lp-fips-build` check again here, which also means that this part does not necessarily have to wait for snapd to complete first.